pkg/datapath/linux/node.go

```diff
@@ -101,7 +101,7 @@ func updateTunnelMapping(oldCIDR, newCIDR *cidr.CIDR, oldIP, newIP net.IP, first
 // cidrNodeMappingUpdateRequired returns true if the change from an old node
 // CIDR and node IP to a new node CIDR and node IP requires to insert/update
 // the new node CIDR.
-func cidrNodeMappingUpdateRequired(oldCIDR, newCIDR *cidr.CIDR, oldIP, newIP net.IP) bool {
+func cidrNodeMappingUpdateRequired(oldCIDR, newCIDR *cidr.CIDR, oldIP, newIP net.IP, oldKey, newKey uint8) bool {
```

Comment:
These are encryption keys, right? I think it'd be useful to make it more clear what these are used for, either by naming them more specifically or extending the function comment to describe what they are.

Reply:
Not actual keys but IDs for Keys so could name *encryptionKeys or encryptionCtx perhaps.